### DISCUSSION TITLE
네트워크 설정 관련 문의입니다

### DISCUSSION THREAD

**Discussion by Qnanana** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#159] [2024년도 기타 질문]
늦게 문의드려 죄송합니다.
우분투 설치 하고 재부팅까지 했으나 영상과 같이 네트워크가 자동으로 뜨지않더라고요. 수동으로 설정해야 할까요??

**Comment by HyeongKeunHong** [comment]
랜카드 드라이버 설치가 되지 않았을 가능성이 높아보입니다.
```
sudo lshw -C network
```
위 명령어를 통해 wifi 모듈명을 알아내신 뒤, 

```
sudo lspci
```
위 명령어를 통해 해당 wifi모듈에 호환이 되는 드라이버를 찾아보시면 됩니다.
랜카드 드라이버가 어떤것인지 알아내신 뒤에는 해당 드라이버를 설치하는 방법을 따로 검색하셔서 해결해야 될 듯 합니다.

만일 어려우시면 유선랜을 통해 인터넷을 사용하는것도 방법이 될 수 있습니다.
